hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -254,7 +245,9 @@ module top_${top["name"]} #(
   ) u_rv_core_ibex (
     // clock and reset
     .clk_i                (main_clk),
-    .rst_ni               (${top["reset_paths"]["sys"]}),
+    // TODO This is necessary right now because there is no way to connect a signal from rv_dm
```

Comment:
does this seem reasonable to you? That whatever fix lands with #1952 will likely fix this one also?

Reply:
#1997 addresses it :)